admin/app/helpers/solidus_admin/components_helper.rb

```diff
@@ -3,7 +3,7 @@
 module SolidusAdmin
   module ComponentsHelper
     def component(name)
-      SolidusAdmin::Config.components[name]
+      SolidusAdmin::Config.components[name].constantize
```

Comment:
@mamhoff This is much needed, thoughts on moving it upstream to the configuration? 

I think it would be better to keep `components[…]` return the constant instead of the string. We can have a a simple object responding to `[]` and `[]=` like we have others to handle constant names in solidus (e.g. `class_name_attribute`).

```rb
class ComponentsRegistry
  def [](key)
    @names[key]&.coonstantize || raise ...
  end

  def []=(key, value)
    @names[key] = value
  end

  def to_h
    @names.map_values { _1.constantize }
  end
end
```

Also I think in non-reloading environment we should start memoizing results.

Reply:
Added a `ComponentsRegistry` class. Because this is in the non-reloadable part of the Engine, I've decided to not add Memoization: It would be very tricky to get right, and I'm not sure the performance gain would actually outweigh the increase in complexity. The [Rails guide](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#reloading-and-stale-objects) even says:
 
 > Bottom line: do not cache reloadable classes or modules.